plugins/auth-backend/src/providers/google/provider.ts

```diff
@@ -53,16 +55,87 @@ export class GoogleAuthProvider
     res: express.Response,
     next: express.NextFunction,
   ) {
-    return passport.authenticate('google', (_, user) => {
-      postMessageResponse(res, {
+    return passport.authenticate('google', (err, user) => {
+      if (err) {
+        return postMessageResponse(res, {
+          type: 'auth-result',
+          error: new Error(`Google auth failed ${err.toString()}`),
```

Comment:
This won't be properly serialized in the response, but I think we can defer the fix for that to the refactor